docs/omnipaxos/communication.md

```diff
@@ -15,7 +15,9 @@ By handling incoming messages and local calls such as `append()`, our local `omn
 
 ```rust
 // send outgoing messages. This should be called periodically, e.g. every ms
-for out_msg in omni_paxos.outgoing_messages() {
+let mut msg_buffer = Vec::new();
```

Comment:
perhaps it's good practice to allocate a size in the example e.g., Vec::with_capacity(1000);